components/item-info.js

```diff
@@ -101,10 +102,12 @@ export default function ItemInfo ({
       </Link>
       <span> \ </span>
       <span>
-        <Link href={`/${item.user.name}`}>
-          @{item.user.name}<span> </span><Hat className='fill-grey' user={item.user} height={12} width={12} />
-          {embellishUser}
-        </Link>
+        <UserPopover text={'@' + item.user.name}>
```

Comment:
@ekzyis in this context it would be possible to pass ```item.user.name``` but on ```text.js``` line ```202``` there's not item.user available

![image](https://github.com/stackernews/stacker.news/assets/241271/d8f9c01c-e315-4fbf-80c1-f7e49f9b7883)

Reply:
Any ideas on how to satisfy both ```text.js``` and ```item-info.js``` without having to pass @username as the text?

Reply:
Mhh, you could remove the `@` outside of `<UserPopover>` since that should be the user name then

Reply:
Off-course 😅

After a day full of bugs my head doesn't work very well hahah... Sorry for bothering you

Reply:
no worries it happens haha